crates/api/src/chronicle_graphql/mod.rs

```diff
@@ -522,6 +524,29 @@ impl SecurityConf {
     }
 }
 
+pub struct Endpoints {
```

Comment:
Perhaps a set of enum values?

Reply:
(That's just me thinking aloud, it's fine if you prefer it as it is, feel free to resolve this.)

Reply:
They are not all mutually exclusive or related, but I agree seeing more than 2 bools at once is a worry. We may be pushing complexity up to configuration here. Simple data / graphql are always reasonable to assume. As they have known base urls, infrastructure can be used to exclude.

It's only health / metrics thats a choice here. Our current requirement is health, we can deal with exposing a prometheus endpoint properly later.

This is my fault, as I thought a prometheus endpoint would be basically free if we based health on metrics - apparently not the case but we will be able to figure it out.

Reply:
Though, I'm happy for none of this to be figured out in this PR if we'd prefer a separate ticket for endpoint conf (or not), at least it's just simple on-off configuration for separable things.

Reply:
This was simply because the compiler was complaining about too many args in a function so I put them in a struct together.